Только в _g3log/g2log: build-release
diff -ru g3log/g2log/CMakeLists.txt _g3log/g2log/CMakeLists.txt
--- g3log/g2log/CMakeLists.txt	2014-09-22 15:54:05.386142576 +0300
+++ _g3log/g2log/CMakeLists.txt	2014-09-22 16:07:20.683263781 +0300
@@ -310,6 +310,13 @@
 endif (USE_G2LOG_UNIT_TEST)
 
 
+FILE (GLOB G3LOG_HEADER_LIST
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/*.hpp
+)
+
+INSTALL(FILES ${G3LOG_HEADER_LIST} DESTINATION include/g3log)
+INSTALL (TARGETS g3logger_shared DESTINATION lib)
+
 
 
 
Только в _g3log/.hg/cache: tags
